subprojects/docs/src/docs/release/notes.md

```diff
@@ -30,6 +30,34 @@ For Java, Groovy, Kotlin and Android compatibility, see the [full compatibility
 
 TBD - load from cache performance improvements and reduced memory consumption for Android builds
 
+## Support kebab case formatting when launching a Gradle build with abbreviated names
+
+When running Gradle builds, you can abbreviate project and task names. For example, you can execute the `compileTest` task by running `gradle cT`.
+
+Until this release, the name abbreviation only worked for camel case names (e.g. `compileTest`). This format is recommended for tasks, but it is unusual for projects. In the Java world the folder names are lower case by convention. On top of that, camel case directory names could cause problems on case-insensitive filesystems.
+
+Many projects - including Gradle - overcome that by using kebab case project directories (e.g. `my-awesome-lib`) and by defining different, camel case project names in the build scripts. This difference means unnecessary extra complexity in the build.
+
+This fixes the issue by adding support for kebab case names in the name matching. Now, you can have following project structure:
+```
```

Comment:
What's the point of showing the whole project structure if we only use one project?

Reply:
You are right, There's no value in that. I've removed the project structure from the text.